crates/admin/protobuf/cluster_ctrl_svc.proto

```diff
@@ -94,6 +94,11 @@ message TrimLogRequest {
   uint64 trim_point = 2;
 }
 
+message TrimLogResponse {
+  uint32 log_id = 1;
+  optional uint64 trim_point = 2;
```

Comment:
Under which condition would a none `trim_point` value a valid response? Maybe document it.